addons/mail/static/src/components/message/message_tests.js

```diff
@@ -983,6 +983,87 @@ QUnit.test('open chat with author on avatar click should be disabled when curren
     );
 });
 
+QUnit.test('Notification Canceled', async function (assert) {
```

Comment:
I know other tests in the file don't have good name, but let's not take this as a reason.

This should describe more what is being tested, if we can consider it that way, at least "basic rendering of canceled notification".

And actually I think this should be a NotificationPopover instead of a Message test? Or all are warranted:
- test in message to ensure the icon is there on cancel
- test in message that click on icon opens popover
- test in notification popover to check its basic rendering

This is ideally though, I would not be against forgetting about this for now if this can help us do more priority tasks.